script/install.sh

```diff
@@ -1197,7 +1197,7 @@ write_env_vars() {
     echo "set _OLD_D_PATH=%PATH%"
     echo "set PATH=${DUB_BIN_PATH:+$(cygpath -w "${DUB_BIN_PATH}");}${winpath};%PATH%"
     if [[ $PROCESSOR_ARCHITECTURE != x86 ]]; then
-        echo "set PATH=${winpath}64;%PATH%"
+        echo "set PATH=${winpath}64;%${winpath};PATH%"
```

Comment:
But, isn't this redundant? The line prepended a value to `PATH`, which already contains `winpath`.

Reply:
Hmm, you are right. Then why wasn't this [1] working? (I don't have a windows machine, I'm just shooting in the dark here)

[1] https://forum.dlang.org/post/nhxzrpzdaltgcaqtyxyh@forum.dlang.org

Reply:
I think the poster there was using the `.exe` installers and not this script.